lib/package/bundleLinter.js

```diff
@@ -179,7 +179,7 @@ var getFormatter = function(format) {
 var executePlugin = function(file, bundle, callback) {
```

Comment:
`excluded` array is not defined here, will produce an error

Reply:
this was missing. I had the wrong code in the original pull request.